styles/Home.module.css

```diff
@@ -25,7 +29,7 @@
   border: none;
 }
 .card {
-  min-width: 20em;
+  /* min-width: 20em; */
```

Comment:
If this is not needed, please remove it instead of commenting it.

Reply:
yes will do that